### PR TITLE
Folder Added:- Assignment2

### PR DESCRIPTION
**Pull request by sandeep597**

### CODE REVIEW

File: SandeepJha/Bootstrap/Assignment 3/Question1.html [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.1/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-F3w7mX95PdgyTmZZMECAngseQB83DfGTowi0iMjiWaeVhAn4FJkqJByhZMI3AhiU" crossorigin="anonymous">
+</head>
```
Comment (Pawan1005) [inline SandeepJha/Bootstrap/Assignment 3/Question1.html:11]: create proper indentation

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

File: SandeepJha/Bootstrap/Assignment 3/Question1.html (hunk 1 above)
Comment (sandeep597) [reply SandeepJha/Bootstrap/Assignment 3/Question1.html:10]: Done.

File: SandeepJha/Bootstrap/Assignment 4/.vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (Pawan1005) [inline SandeepJha/Bootstrap/Assignment 4/.vscode/settings.json:3]: remove this file

File: SandeepJha/Bootstrap/Assignment 4/Question1.html [hunk 3]
```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+</head>
```
Comment (Pawan1005) [inline SandeepJha/Bootstrap/Assignment 4/Question1.html:10]: do proper indentation for head part

File: SandeepJha/Bootstrap/Assignment 5/Question1.html [hunk 4]
```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+</head>
```
Comment (Pawan1005) [inline SandeepJha/Bootstrap/Assignment 5/Question1.html:9]: do proper indentation for head part

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

File: SandeepJha/Bootstrap/Assignment 4/Question1.html (hunk 3 above)
Comment (sandeep597) [reply SandeepJha/Bootstrap/Assignment 4/Question1.html]: done

File: SandeepJha/Bootstrap/Assignment 4/.vscode/settings.json (hunk 2 above)
Comment (sandeep597) [reply SandeepJha/Bootstrap/Assignment 4/.vscode/settings.json]: done

**Pull request merged**